check_ldap_monitor_389ds.pl

```diff
@@ -0,0 +1,607 @@
+#! /usr/bin/perl -w
+
+#==========================================================================
+# Summary
+#==========================================================================
+# Check LDAP query
```

Comment:
Change the summary here to "Check 389DS monitor"

Reply:
Done 7521371daa8421031c85df0c6884f97303a42757